tests/opencl_interop/opencl_interop_constructors.cpp

```diff
@@ -147,54 +149,70 @@ class TEST_NAME : public sycl_cts::util::test_base_opencl {
       /** check program (context, cl_program) constructor
        */
       {
+        cl_program clProgram = nullptr;
```

Comment:
Is it standard OpenCL to assume that a `cl_program` is a pointer, by the way?

Reply:
`cl_program` is defined as a [pointer](https://github.com/KhronosGroup/OpenCL-Headers/blob/master/CL/cl.h#L46) in OpenCL-Headers.
I don't know better way to initialize `cl_program` object, so I open to suggestions.

Reply:
Possible suggestions:
- keep your solution because this is a well-known implementation that leaks to all the implementation because of ICD;
- improve the OpenCL specification to introduce initializers in the standard (probably required by things like AUTOSAR/MISRA/...);
- do not use such an unsafe API :-)

So probably your solution is good to have things done... :-)

Reply:
Actually C++ provides a portable way to do it:
```
cl_program clProgram {};
```

Reply:
I applied this suggestion in https://github.com/KhronosGroup/SYCL-CTS/pull/12/commits/aab8c8dfe4903455937dad2b597e1d1eeb220d95.